packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,434 @@
+
+##### Keyboard navigation
+
+The Form Pill Group is focusable, but only one pill is focusable at a time. This means the Pill Group is a single tab stop to a keyboard user. Once focused within the Pill Group the following keyboard interactions apply.
+
+- Left and Right arrow keys move focus within the group
+- If selectable, spacebar and enter will toggle pill selection
+- If a pill has a supplied action via `onClick`, enter key will trigger that action
+- If dismissable, a Pill can be removed by pressing the delete or backspace key
+
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/1bea986ec6a24cd494950e2c12f10562"
+      frameborder="0"
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
+
+#### Display Pill vs Form Pill
+
+There are some very important semantic differences between a Display Pill and Form Pill group. Below is a table explaining the main differences to get a better understanding of when to use each type of Pill.
+
+<FormPillVsDisplayPillTable />
+
+## Examples
+
+### Basic Form Pill Group
+
+The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+
+This will provide you with some internal state management and keyboard navigation.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Selectable Form Pill Group
+
+Outside of keyboard navigation, Form Pills are presentational. If you would like to make the Pill Group provide selection options, you can manage the state of those pills and seemlessly merge it with the state return from the `useFormPillState` hook.
+
+Track which pill is selected in a separate store of state. When rendering the Pill Group, cross reference the pills with the selected state, and conditoinally set `selected` on each `FormPill` that requires it.
+
+The `onSelect` event handler will fire when ever the pill is clicked, or the spacebar or enter key is pressed. Use this to respond to your users selection interactions.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {selectableExample}
+</LivePreview>
+
+### Removable Form Pill Group
+
+For removable Form Pills, because the Form Pill Group is largely presentational, you can provide removable functionality by managing the state of the rendered Pills. By responding to user interactions, hooking into the `onDismiss` event handler, you are able to update the rendered state of the Form Pill Group and selectively remove pills from the collection.
+
+The `onDismiss` event handler will fire when a user clicks on the close x, or presses their backspace or delete key when foused on a pill.
```

Comment:
I think it's worth writing explicitly that Form Pills become removable when you give them an onDismiss handler
```suggestion
Form Pills are given a closing x button when provided an onDismiss event handler. Because the Form Pill Group is largely presentational, you can provide "removable" functionality by managing the state of the rendered Pills. By responding to user interactions and hooking into the `onDismiss` event handler, the rendered state of the Form Pill Group can be updated and pills can be selectively removed from the collection.

The `onDismiss` event handler will fire when a user clicks on the close x, or presses their backspace or delete key when foused on a pill.
```